sample/src/main/kotlin/com/slack/circuit/sample/petlist/AboutScreen.kt

```diff
@@ -44,7 +44,7 @@ import kotlinx.coroutines.flow.Flow
 import kotlinx.parcelize.Parcelize
 
 @Parcelize
-class AboutScreen : Screen {
+object AboutScreen : Screen {
```

Comment:
For my understanding, is this being changed from a `class` to an `object` because it's more semantically correct?

Reply:
yep! No data => just make it an object